src/pages/dashboard/OperatorHbarChart/operatorsHbarChart.scss

```diff
@@ -0,0 +1,57 @@
+.operator {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: 31px;
+    position: relative;
+
+    .operatorName {
+        width: 16%;
+        border-left: 1px solid white;
+        color: white;
+        font-size: 16px;
+        font-weight: 700;
+        display: flex;
+        align-items: center;
+        padding-right: 20px;
+        box-sizing: border-box;
+    }
+    .operatorBar{
+        flex-grow: 1;
+
+        .operatorBarTotal {
+            background-image: linear-gradient(to bottom, transparent 49%, black 49%, black 51%, transparent 51%);
+            
+            .operatorBarActual {
+                text-align: left;
+                font-size: 28px;
+                font-weight: 500;
+                color: white;
+                padding: 10px;
+            }
+        }
+    }
+
+    .tooltip {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(0,0,0,0.5);
```

Comment:
let's move colors to variables or consts, so they can be easily tracked and reused

Reply:
done